app/views/stories/_listdetail.html.erb

```diff
@@ -153,6 +153,10 @@ class="story <%= story.current_upvoted? ? "upvoted" : "" %>
             <span> | </span><%= link_to "hide", story_hide_path(story.short_id),
               :class => "hider" %>
           <% end %>
+          <% if story.is_disownable_by_user?(@user) %>
+          |
+          <a tabindex="0" class="story-disowner" href="#">disown</a>
```

Comment:
```suggestion
          <a class="story-disowner" href="#">disown</a>
```

This is unneeded. I see where you copied it from, I'll clean up the other templates.